cmd/eval-dev-quality/cmd/evaluate_test.go

```diff
@@ -305,6 +314,8 @@ func TestEvaluateExecute(t *testing.T) {
 	})
 	t.Run("Model filter", func(t *testing.T) {
 		t.Run("openrouter.ai", func(t *testing.T) {
+			t.Setenv("OLLAMA_MODELS", t.TempDir()) // Set the Ollama model directory to something else so even if the user executing these tests has Ollama installed, we don't query their models.
```

Comment:
I think, we want that. I do not want to query all models again that are run in the tests. So by default, i think we should use the cached models. Removing this.